packages/backend/src/plugins/techdocs.ts

```diff
@@ -49,8 +46,17 @@ export default async function createPlugin({
   const urlPreparer = new UrlPreparer(reader, logger);
   preparers.register('url', urlPreparer);
 
+  // Generators are used for generating documentation sites.
+  const generators = new Generators();
+  const techdocsGenerator = new TechdocsGenerator(logger, config);
+  generators.register('techdocs', techdocsGenerator);
+
+  // Publishers are used for
+  // 1. Publishing generated files to storage
+  // 2. Fetching files from storage and passing them to TechDocs frontend.
```

Comment:
These docs only end up in this repo and aren't too visible, want to add them to the create-app template as well?

Best do in a followup PR tho, needs a proper changeset and so on ^_>

Reply:
Good point!